sqlite/import.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Do you mean to checkin the datacommons.db?

Reply:
Yea, I want to expand that and use it for testing; Also users would use it as "hello world" db.